core/src/tile/tileManager.h

```diff
@@ -26,6 +27,10 @@ class View;
  */
 class TileManager {
 
+    const static size_t MAX_WORKERS = 2;
+    const static size_t MAX_DOWNLOADS = 4;
+    const static size_t DEFAULT_CACHE_SIZE = 32*1024*1024; // 32 MB
```

Comment:
I misunderstood a part of the process, stepping through with debug answered my question!